src/utils/internal/getCallFrame.ts

```diff
@@ -2,7 +2,8 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  const frames: string[] = (new Error().stack as string).split('\n')
+  // In <IE11, new Error may return an undefined stack
+  const frames: string[] = ((new Error().stack || '') as string).split('\n')
```

Comment:
This line is getting quite complex, what do you think if we move it into its own variable?

```ts
const stack = (new Error().stack || '') as string
const frames: string[] = stack.split('\n')
```

It's subjective, but I find it to read much better.

Reply:
I actually had it this way first, adjusted in 895af0f05cbd9f2667ed02dc4622e026bd15299f.